### PR TITLE
{bio}[foss/2015b] ClustalW2 2.1 (REVIEW)

### PR DESCRIPTION
**Pull request by nathanhaigh**

### CODE REVIEW

File: easybuild/easyconfigs/c/ClustalW2/ClustalW2-2.1-foss-2015b.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+version = '2.1'
+
+homepage = 'http://www.ebi.ac.uk/Tools/msa/clustalw2/'
+description = """ClustalW2 is a general purpose multiple sequence alignment program for DNA or proteins."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+source_urls = ['http://www.clustal.org/download/%(version)s']
+sources = ['%s-%s.tar.gz' % (name[:-1].lower(), version)]
```
Comment (boegel) [inline easybuild/easyconfigs/c/ClustalW2/ClustalW2-2.1-foss-2015b.eb]: let's change this to:

```python
sources = ['ClustalW2-%(version)s.tar.gz']
```

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/5af1e5666c11e7192da8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/ac8518225af82eaea3b6 for a full test report.

**Comment by nathanhaigh** [comment]
@boegel I'm wondering if this should be renamed from `ClustalW2` to `ClustalW`. What's the convension for using part of the tool's version string in the name? 

Maybe to answer my own question somewhat: Having a different name, by appending part of the version string, allows both tools to be loaded in the environment at the same time. But what if they both have the same named executables, which one wins when trying to run one?

**Comment by boegel** [comment]
@nathanhaigh: we're just going with the 'official' software name usally, cfr. http://www.ebi.ac.uk/Tools/msa/clustalw2 in this case

There are other cases too, Qt5 vs Qt is a recent one. In some situations, you need both Qt 4.x and Qt5 in the same software stack, so you need to have separate names in that case as you've mentioned, indeed.

**Comment by boegel** [comment]
lgtm

**Pull request merged**